README.md

```diff
@@ -8,37 +8,29 @@ This is a [Cluster API](https://cluster-api.sigs.k8s.io/introduction.html) provi
 # Installation instructions
 
 Prerequisites:
-- [clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) (v1.1.5+)
+- [clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) (v1.7.4+)
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 - A Kubernetes cluster where you will have cluster-admin permissions
 - Optional, depending on how you expose the vcluster instance - [vcluster CLI](https://www.vcluster.com/docs/getting-started/setup)
 
-Install the vcluster provider (latest stable release)
+Install the vcluster provider
 
 ```shell
 clusterctl init --infrastructure vcluster
 ```
 
-or for the latest alpha release (which uses the v0.20 [vcluster.yaml](https://www.vcluster.com/docs/vcluster/configure/vcluster-yaml/) format per default):
-
-```shell
-clusterctl init --infrastructure vcluster:v0.2.0-alpha.2
-```
-
 Next you will generate a manifest file for a vcluster instance and create it in the management cluster.
 Cluster instance is configured using clusterctl parameters and environment variables - CHART_NAME, CHART_REPO, CHART_VERSION, VCLUSTER_HOST and VCLUSTER_PORT.
 In the example commands below, the HELM_VALUES variable will be populated with the contents of the `values.yaml` file.
 ```shell
 export CLUSTER_NAME=vcluster
 export CLUSTER_NAMESPACE=vcluster
-export KUBERNETES_VERSION=1.28.9
```

Comment:
Do we no longer support specifying a k8s version? What k8s version is used if we don't specify it? 

Is it the default k8s version in the vcluster version?

Reply:
Yes. The version of vcluster itself can be chosen by the user via the `CHART_VERSION` env var.